spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -151,7 +151,7 @@
         allow(user).to receive(:establishing_in_person_enrollment).and_return(nil)
       end
 
-      it 'disables double address verification for the user' do
+      it 'indicates to the IDV agent that ipp_enrollment_in_progress is disabled' do
```

Comment:
i think rather than disabled/enabled we should change the wording to true/false or something more in line with the fact that the user is going through ipp.

Reply:
Ok, yeah I guess the flow isn't being altered anymore, right? We're just monitoring whether their in the middle of an enrollment, right?

Reply:
yes